pkg/rime/rime_test.go

```diff
@@ -0,0 +1,131 @@
+package rime
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestToString(t *testing.T) {
+	data := []byte("Some Data")
+	p := &Payload{
+		Type:      StringData,
+		Length:    uint16(len(data)),
+		Timestamp: uint32(1618690395),
+		Data:      data,
+	}
+
+	actual := p.ToString()
+
+	expected := "[TYP: 1 | LEN: 9 | TIM: 2021-04-17 23:13:15 +0300 EEST | DAT: Some Data]"
+	if expected != actual {
+		t.Errorf("Wrong value. Expected: %v, actual: %v", expected, actual)
+	}
+}
+
+func TestWrite(t *testing.T) {
+	buf := new(bytes.Buffer)
+	w := NewWriter(buf)
+	data := []byte("Some Data")
+	p := &Payload{
+		Type:      StringData,
+		Length:    uint16(len(data)),
+		Timestamp: uint32(1618690395),
+		Data:      data,
+	}
+
+	err := w.Write(p)
+	if err != nil {
+		t.Errorf("Error: " + err.Error())

```

Comment:
This could be simpler.

```suggestion
		t.Errorf("Error: %v", err)
```